src/Core/Bases/A11yProperty.cs

```diff
@@ -159,7 +159,7 @@ private string GetBoundingRectangleText()
             if ((double)arr[2] < 0 || (double)arr[3] < 0)
             {
                 // the 3rd and 4th values in array are negative value, we need to show value in different format like l,t,w,h
-                text = string.Format(CultureInfo.InvariantCulture, "[l={0},t={1},w={2},h={3}]", arr[0], arr[1], arr[2], arr[3]);
+                text = ExtensionMethods.WithParameters("[l={0},t={1},w={2},h={3}]", arr[0], arr[1], arr[2], arr[3]);
```

Comment:
This string gets included in what gets returned to the scan caller (you can test this by running the CLI in verbose mode). As such, this might be a string to localize

Reply:
I'll factor this out into a new resource then, and it sounds like we will have to add this to the list of projects to localize. There are several other short strings that I'm seeing in this project that I had assumed wouldn't be displayed to the user, so I hadn't prioritized but it sounds like we should reconsider that. Do you know if any of these are exposed to the user?

https://github.com/microsoft/axe-windows/blob/main/src/Core/Bases/A11yProperty.cs#L101
https://github.com/microsoft/axe-windows/blob/main/src/Core/CustomObjects/Converters/EnumTypeConverter.cs#L22
https://github.com/microsoft/axe-windows/blob/main/src/Core/CustomObjects/CustomProperty.cs#L28

Reply:
I would guess that these are all in scope for loc

Reply:
Okay, I'll make a separate PR to extract these out into a resx file then, and add the core project to our list of projects to be localized